docs/in-depth/parameters.md

```diff
@@ -0,0 +1,119 @@
+## Defining parameters
+
+In MoMEMta, once the `ConfigurationReader` has been "frozen" to give a `Configuration` object, it is not possible anymore to modify anything in the integration. However, some parameters can be modified in the `ConfigurationReader` from the C++ code (allowing e.g. to carry out mass scans easily). In order to do that, the parameters have to be put in the special `parameters` table in Lua:
+```Lua
+parameters = {
+  my_mass = 173.,
+  my_width = 2.5
+}
+```
+These parameters can then be linked with the modules' parameters using the `parameter()` Lua function:
+```Lua
+BreitWignerGenerator.m_prop = {
+  ps_point = add_dimension(),
+  mass = parameter('my_mass'), -- Access the value through the "parameter" function, allowing it to be modified from the C++ code
+  width = parameters.my_width -- Access the value as a regular Lua table. It will NOT be possible to modify it later on
```

Comment:
It can be worth IMO to add a `!!! danger` block below insisting on this fact: you have to pass through the `parameter` function if you want the parameter to be editable